### PR TITLE
Add fields

### LINKED ISSUE #39: Send additional fields back with social posts
@ericljpemberton needs the content of the post returned after a post.

Posts should also have the profile photo url returned as part of the post.

### PR DESCRIPTION
**Pull request by kyl191**
Fixes #39.

Returned posts now have the profile picture pulled from the DB instead of assuming a fixed location.

And added the text + media link for @ericljpemberton

### CODE REVIEW

File: api/models/social.py [hunk 1]
```diff
@@ -65,9 +65,10 @@ def format_post_response(posts, req_userid):
     for item in posts:
         item['timestamp'] = str(item.get('timestamp'))
         item['id'] = item['timestamp']
-        item['poster_name'] = res.get(item['poster_id'], 'Random User')
-        item['poster_icon'] = 'https://{}.s3.amazonaws.com/profilepic-{}.jpg'\
-            .format(PROFILE_PIC_BCKT, item['poster_id'])
+        item['poster_name'] = res.get(item['poster_id'], {}).\
+            get('name', "Random User")
+        item['poster_icon'] = res.get(item['poster_id'], {}).\
+            get('poster_icon')
```
Comment (TheKevJames) [inline api/models/social.py]: Default icon?

Comment (kyl191) [reply api/models/social.py]: What's the link?

Comment (TheKevJames) [reply api/models/social.py]: Throw this in a bucket and use that: [icon](http://donatered-asset.s3.amazonaws.com/assets/default/default_user-884fcb1a70325256218e78500533affb.jpg)

Comment (TheKevJames) [reply api/models/social.py]: Kidding, [use this](https://eliaslealblog.files.wordpress.com/2014/03/user-200.png)

**Review by TheKevJames — APPROVED** [review]

**Pull request merged**